StripePayments/StripePayments/Source/API Bindings/STPAPIClient+LinkAccountSession.swift

```diff
@@ -75,6 +80,7 @@ public extension STPAPIClient {
         parameters["amount"] = amount
         parameters["currency"] = currency
         parameters["on_behalf_of"] = onBehalfOf
+        parameters["link_mode"] = linkMode?.rawValue ?? "LINK_DISABLED"
```

Comment:
Do we need to pass `LINK_DISABLED`, or could we only pass the `link_mode` parameter when it's non-nil?

```swift
if let linkMode {
    parameters["link_mode"] = linkMode.rawValue
}
```

Reply:
Yep, we want to pass `LINK_DISABLED` to be in line with Web’s behavior.